alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultClientCertificateKeyAccessor.java

```diff
@@ -0,0 +1,88 @@
+package com.synopsys.integration.alert.database.api;
+
+import java.time.OffsetDateTime;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.alert.api.common.model.exception.AlertConfigurationException;
+import com.synopsys.integration.alert.common.persistence.accessor.UniqueConfigurationAccessor;
+import com.synopsys.integration.alert.common.persistence.model.ClientCertificateKeyModel;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.security.EncryptionUtility;
+import com.synopsys.integration.alert.common.util.DateUtils;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateKeyEntity;
+import com.synopsys.integration.alert.database.certificates.ClientCertificateKeyRepository;
+
+public class DefaultClientCertificateKeyAccessor implements UniqueConfigurationAccessor<ClientCertificateKeyModel> {
```

Comment:
A general point to this accessor, I don't believe we should be using the AlertRestConstants.DEFAULT_CLIENT_CERTIFICATE_KEY_NAME for the name field of ClientCertificateKeyModel. Alias is specific to the content which is in the other model object. For this my thoughts are that we should perhaps use the AlertRestConstants.DEFAULT_CONFIGURATION_NAME

Reply:
Change DEFAULT_CLIENT_CERTIFICATE_KEY_NAME only